AudioKit/Common/Playgrounds/Basics.playground/contents.xcplayground

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<playground version='6.0' target-platform='ios' display-mode='raw'>
+<playground version='6.0' target-platform='ios' display-mode='rendered'>
```

Comment:
It's annoying that the display-mode is saved - however, as a default I think it should be rendered. A bit easier for first-time users!